[2025-04-08T23:24:54.762+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-08T23:24:54.779+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.run_dbt_staging manual__2025-04-08T23:24:40.295539+00:00 [queued]>
[2025-04-08T23:24:54.788+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.run_dbt_staging manual__2025-04-08T23:24:40.295539+00:00 [queued]>
[2025-04-08T23:24:54.789+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-08T23:24:54.969+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): run_dbt_staging> on 2025-04-08 23:24:40.295539+00:00
[2025-04-08T23:24:54.976+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1113) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-08T23:24:54.978+0000] {standard_task_runner.py:72} INFO - Started process 1115 to run task
[2025-04-08T23:24:54.978+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'run_dbt_staging', 'manual__2025-04-08T23:24:40.295539+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/driven_data_pipeline.py', '--cfg-path', '/tmp/tmpx74scw_w']
[2025-04-08T23:24:54.979+0000] {standard_task_runner.py:105} INFO - Job 44: Subtask run_dbt_staging
[2025-04-08T23:24:55.142+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.run_dbt_staging manual__2025-04-08T23:24:40.295539+00:00 [running]> on host 5a8aeda502d0
[2025-04-08T23:24:55.592+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='run_dbt_staging' AIRFLOW_CTX_EXECUTION_DATE='2025-04-08T23:24:40.295539+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-08T23:24:40.295539+00:00'
[2025-04-08T23:24:55.593+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-08T23:24:55.606+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-08T23:24:55.608+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'set -x; cd /opt/***/dbt && dbt run --select tag:staging']
[2025-04-08T23:24:55.617+0000] {subprocess.py:86} INFO - Output:
[2025-04-08T23:24:55.618+0000] {subprocess.py:93} INFO - + cd /opt/***/dbt
[2025-04-08T23:24:55.619+0000] {subprocess.py:93} INFO - + dbt run --select tag:staging
[2025-04-08T23:24:59.172+0000] {subprocess.py:93} INFO - [0m23:24:59  Running with dbt=1.8.0
[2025-04-08T23:24:59.808+0000] {subprocess.py:93} INFO - [0m23:24:59  Registered adapter: postgres=1.8.2
[2025-04-08T23:24:59.973+0000] {subprocess.py:93} INFO - [0m23:24:59  Unable to do partial parsing because saved manifest not found. Starting full parse.
[2025-04-08T23:25:00.127+0000] {subprocess.py:93} INFO - [0m23:25:00  Encountered an error:
[2025-04-08T23:25:00.128+0000] {subprocess.py:93} INFO - 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte
[2025-04-08T23:25:00.130+0000] {subprocess.py:93} INFO - [0m23:25:00  Traceback (most recent call last):
[2025-04-08T23:25:00.131+0000] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.12/site-packages/dbt/cli/requires.py", line 138, in wrapper
[2025-04-08T23:25:00.131+0000] {subprocess.py:93} INFO -     result, success = func(*args, **kwargs)
[2025-04-08T23:25:00.132+0000] {subprocess.py:93} INFO -                       ^^^^^^^^^^^^^^^^^^^^^
[2025-04-08T23:25:00.132+0000] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.12/site-packages/dbt/cli/requires.py", line 101, in wrapper
[2025-04-08T23:25:00.133+0000] {subprocess.py:93} INFO -     return func(*args, **kwargs)
[2025-04-08T23:25:00.133+0000] {subprocess.py:93} INFO -            ^^^^^^^^^^^^^^^^^^^^^
[2025-04-08T23:25:00.134+0000] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.12/site-packages/dbt/cli/requires.py", line 218, in wrapper
[2025-04-08T23:25:00.205+0000] {subprocess.py:93} INFO -     return func(*args, **kwargs)
[2025-04-08T23:25:00.206+0000] {subprocess.py:93} INFO -            ^^^^^^^^^^^^^^^^^^^^^
[2025-04-08T23:25:00.206+0000] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.12/site-packages/dbt/cli/requires.py", line 247, in wrapper
[2025-04-08T23:25:00.207+0000] {subprocess.py:93} INFO -     return func(*args, **kwargs)
[2025-04-08T23:25:00.207+0000] {subprocess.py:93} INFO -            ^^^^^^^^^^^^^^^^^^^^^
[2025-04-08T23:25:00.208+0000] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.12/site-packages/dbt/cli/requires.py", line 294, in wrapper
[2025-04-08T23:25:00.208+0000] {subprocess.py:93} INFO -     return func(*args, **kwargs)
[2025-04-08T23:25:00.209+0000] {subprocess.py:93} INFO -            ^^^^^^^^^^^^^^^^^^^^^
[2025-04-08T23:25:00.209+0000] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.12/site-packages/dbt/cli/requires.py", line 320, in wrapper
[2025-04-08T23:25:00.210+0000] {subprocess.py:93} INFO -     ctx.obj["manifest"] = parse_manifest(
[2025-04-08T23:25:00.210+0000] {subprocess.py:93} INFO -                           ^^^^^^^^^^^^^^^
[2025-04-08T23:25:00.211+0000] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.12/site-packages/dbt/parser/manifest.py", line 1898, in parse_manifest
[2025-04-08T23:25:00.211+0000] {subprocess.py:93} INFO -     manifest = ManifestLoader.get_full_manifest(
[2025-04-08T23:25:00.212+0000] {subprocess.py:93} INFO -                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2025-04-08T23:25:00.212+0000] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.12/site-packages/dbt/parser/manifest.py", line 330, in get_full_manifest
[2025-04-08T23:25:00.213+0000] {subprocess.py:93} INFO -     manifest = loader.load()
[2025-04-08T23:25:00.213+0000] {subprocess.py:93} INFO -                ^^^^^^^^^^^^^
[2025-04-08T23:25:00.213+0000] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.12/site-packages/dbt/parser/manifest.py", line 377, in load
[2025-04-08T23:25:00.214+0000] {subprocess.py:93} INFO -     file_reader.read_files()
[2025-04-08T23:25:00.214+0000] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.12/site-packages/dbt/parser/read_files.py", line 210, in read_files
[2025-04-08T23:25:00.215+0000] {subprocess.py:93} INFO -     self.read_files_for_project(project, file_types)
[2025-04-08T23:25:00.215+0000] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.12/site-packages/dbt/parser/read_files.py", line 217, in read_files_for_project
[2025-04-08T23:25:00.216+0000] {subprocess.py:93} INFO -     project_files[file_type_info["parser"]] = read_files_for_parser(
[2025-04-08T23:25:00.216+0000] {subprocess.py:93} INFO -                                               ^^^^^^^^^^^^^^^^^^^^^^
[2025-04-08T23:25:00.217+0000] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.12/site-packages/dbt/parser/read_files.py", line 165, in read_files_for_parser
[2025-04-08T23:25:00.217+0000] {subprocess.py:93} INFO -     source_files = get_source_files(
[2025-04-08T23:25:00.218+0000] {subprocess.py:93} INFO -                    ^^^^^^^^^^^^^^^^^
[2025-04-08T23:25:00.218+0000] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.12/site-packages/dbt/parser/read_files.py", line 154, in get_source_files
[2025-04-08T23:25:00.218+0000] {subprocess.py:93} INFO -     file = load_source_file(fp, parse_file_type, project.project_name, saved_files)
[2025-04-08T23:25:00.219+0000] {subprocess.py:93} INFO -            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2025-04-08T23:25:00.220+0000] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.12/site-packages/dbt/parser/read_files.py", line 82, in load_source_file
[2025-04-08T23:25:00.220+0000] {subprocess.py:93} INFO -     file_contents = load_file_contents(path.absolute_path, strip=True)
[2025-04-08T23:25:00.221+0000] {subprocess.py:93} INFO -                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2025-04-08T23:25:00.221+0000] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.12/site-packages/dbt_common/clients/system.py", line 178, in load_file_contents
[2025-04-08T23:25:00.221+0000] {subprocess.py:93} INFO -     to_return = handle.read().decode("utf-8")
[2025-04-08T23:25:00.222+0000] {subprocess.py:93} INFO -                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2025-04-08T23:25:00.222+0000] {subprocess.py:93} INFO - UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte
[2025-04-08T23:25:00.223+0000] {subprocess.py:93} INFO - 
[2025-04-08T23:25:00.926+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2025-04-08T23:25:01.010+0000] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/bash.py", line 249, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2025-04-08T23:25:01.014+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=extract_raw_data_pipeline, task_id=run_dbt_staging, run_id=manual__2025-04-08T23:24:40.295539+00:00, execution_date=20250408T232440, start_date=20250408T232454, end_date=20250408T232501
[2025-04-08T23:25:01.134+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-08T23:25:01.135+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 44 for task run_dbt_staging (Bash command failed. The command returned a non-zero exit code 2.; 1115)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/bash.py", line 249, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2025-04-08T23:25:01.161+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-04-08T23:25:01.181+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-08T23:25:01.214+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
